src/renderers/shared/utils/CallbackQueue.js

```diff
@@ -101,6 +102,4 @@ Object.assign(CallbackQueue.prototype, {
 
 });
 
-PooledClass.addPoolingTo(CallbackQueue);
-
-module.exports = CallbackQueue;
+module.exports = PooledClass.addPoolingTo(CallbackQueue);
```

Comment:
Do we need to do it in a single statement here? I’m asking because usually we write `module.exports = ThingWithSameNameAsFile` and the inconsistency is slightly jarring.